README.md

```diff
@@ -50,9 +50,10 @@ The following commands install the development version of The Lounge:
 git clone https://github.com/thelounge/lounge.git
 cd lounge
 npm install
-npm start
```

Comment:
I wouldn't remove this line, and your next change could only be about running subcommands.